random_question/weighted_random.py

```diff
@@ -10,11 +10,26 @@
 
 # TODO: Hess should implement!
 def weighted_sampler(data, weights):
-    """Should return a random item from data array, weighted by the weights array"""
-    raise NotImplementedError("Hess should implement!")
+    random_number=random.random()
+    print('this is the:')
+    print(random_number)
+    Dfunction = arrange_array(weights)
```

Comment:
both the function and the variable have weird names. names should be super-descriptive. Naming is one of the most important things in coding. it's how you make your code readable.

the function could possibly be called `accumulate_array`. the result is maybe an `accumulated_array`.
Actually, in python lingo its a `list`, not an array! an array is a different data structure in python

Reply:
DONE